### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by statisticals**
add test file for mk_reactable function

### CODE REVIEW

File: tests/testthat/test-mk_reactable.R [hunk 1]
```diff
@@ -0,0 +1,36 @@
+test_that("mk_reactable() handles errors correctly about missing data ", {
```
Comment (wangben718) [inline tests/testthat/test-mk_reactable.R:1]: library(reactable)
devtools::load_all()

Comment (wangben718) [reply tests/testthat/test-mk_reactable.R:1]: Need to load package before test

Comment (wangben718) [reply tests/testthat/test-mk_reactable.R:1]: also load data if needed: 
data(adsl)
data(adae)

File: tests/testthat/test-mk_reactable.R [hunk 2]
```diff
@@ -0,0 +1,41 @@
+library(reactable)
+devtools::load_all()
```
Comment (elong0527) [inline tests/testthat/test-mk_reactable.R:2]: please change this to `library(forestly)` 

we shall avoid `load_all` in formal script.

**Pull request merged**